install_config/aggregate_logging.adoc

```diff
@@ -427,15 +427,18 @@ must be at least 512M. Possible suffixes are G,g,M,m.
 |A key/value map added to a PVC in order to select specific PVs.
 
 |`openshift_logging_es_pvc_dynamic`
-|Set to `true` to have PVC claims annotated so that their backing storage is
-dynamically provisioned. When set to `true` the storage class is ignored and not
-set to the PVC.
+|Set to `true` so the backing storage is dynamically provisioned. When set to
```

Comment:
s/Set to `true` so the backing storage is dynamically provisioned./ To dynamically provision the backing storage, set the parameter value to \`true`.